src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -166,14 +178,10 @@ public void setAuthorList(List<Author> authorList) {
     /**
      * Clears author mapping information and resets it with the details of current author list.
      */
-    public void resetAuthorInformation(List<String> ignoreGlobList) {
```

Comment:
`removeIgnoredAuthors` invokes this method but does not need `propogateIgnoreGlobList`. In addition, `propogateIgnoreGlobList` does not seem to fit the purpose of this method so I moved it out.